GPT/gpt.talon

```diff
@@ -5,10 +5,10 @@ model help$: user.gpt_help()
 #   Example: `model fix grammar below` -> Fixes the grammar of the selected text and pastes below
 #   Example: `model explain this` -> Explains the selected text and pastes in place
 #   Example: `model fix grammar clip to browser` -> Fixes the grammar of the text on the clipboard and opens in browser`
-model <user.modelPrompt> [{user.modelSource}] [{user.modelDestination}]:
+model <user.modelPrompt> [{user.modelSource}] ([{user.modelPasteDestination}] [{user.modelInsertionModifier}] | [{user.modelDestination}]):
```

Comment:
what is the difference between a paste destination, an insertion modifier, and a model destination?

Reply:
Yeah I don't particularly want to have an additional list for paste destination in addition to the model destination. The issue is that whenever we split things into separate lists we have to check more permutations to make sure they are valid (ie we can't paste above in a window or with verbal output) and paste destination doesn't really strike me as semantically different enough to warrant its own list. Unless I am missing something

Reply:
I could also see an argument that "below" and "above" are insertion modifiers as well-what if we were to put those in the same talon list and allow multiple to be applied?